torchgeo/datasets/utils.py

```diff
@@ -431,3 +436,26 @@ def sort_sentinel2_bands(x: str) -> str:
     if x == "B8A":
         x = "B08A"
     return x
+
+
+def draw_semantic_segmentation_masks(
+    image: Tensor, mask: Tensor, alpha: float = 0.5, colors: Optional[ColorMap] = None
+) -> np.ndarray:  # type: ignore[type-arg]
+    """Overlay a semantic segmentation mask onto an image.
+
+    Args:
+        image: tensor of shape (3, h, w)
+        mask: tensor of shape (h, w) with pixel values representing the classes
+        alpha: alpha blend factor
+        colors: (Optional) list of RGB int tuples, or color strings e.g. red, #FF00FF
```

Comment:
```suggestion
        colors: list of RGB int tuples, or color strings e.g. red, #FF00FF
```
It will show up as optional in the docs due to typing